Looks like there is a py3 issue with the JSON serialization: 
https://travis-ci.org/choderalab/assaytools/jobs/228916368#L671-L688

The failure seems to be caused by the `json.dump` of the `metadata`:
```
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test/bin/quickmodel", line 11, in <module>
    load_entry_point('assaytools==0.2.0', 'console_scripts', 'quickmodel')()
  File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/assaytools/scripts/quickmodel.py", line 241, in entry_point
    quick_model(inputs, nsamples=args.nsamples, nthin=args.nthin)
  File "/home/travis/miniconda/envs/test/lib/python3.4/site-packages/assaytools/scripts/quickmodel.py", line 211, in quick_model
    json.dump(metadata, outfile, sort_keys = True, indent = 4, ensure_ascii=False)
  File "/home/travis/miniconda/envs/test/lib/python3.4/json/__init__.py", line 178, in dump
    for chunk in iterable:
  File "/home/travis/miniconda/envs/test/lib/python3.4/json/encoder.py", line 422, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/home/travis/miniconda/envs/test/lib/python3.4/json/encoder.py", line 396, in _iterencode_dict
    yield from chunks
  File "/home/travis/miniconda/envs/test/lib/python3.4/json/encoder.py", line 429, in _iterencode
    o = _default(o)
  File "/home/travis/miniconda/envs/test/lib/python3.4/json/encoder.py", line 173, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: 0 is not JSON serializable
```

--- a/scripts/quickmodel.py
+++ b/scripts/quickmodel.py
@@ -179,6 +179,7 @@ def quick_model(inputs, nsamples=1000, nthin=20):
             else:
                 Kd_summary = "%.3e M +- %.3e M" % (Kd, dKd)
 
+
             outputs = {
                 #'raw_data_file'   : my_file,
                 'complex_fluorescence' : complex_fluorescence[name],
@@ -201,6 +202,7 @@ def quick_model(inputs, nsamples=1000, nthin=20):
 
             metadata['ligand_fluorescence'] = metadata['ligand_fluorescence'].tolist()
             metadata['complex_fluorescence'] = metadata['complex_fluorescence'].tolist()
+            metadata['t_equil'] = metadata['t_equil'].tolist()
             metadata['bin_edges'] = metadata['bin_edges'].tolist()
             metadata['hist'] = metadata['hist'].tolist()
             metadata['Pstated'] = metadata['Pstated'].tolist()

--- a/scripts/trace_plots.py
+++ b/scripts/trace_plots.py
@@ -1,0 +1,53 @@
+import matplotlib
+matplotlib.use('Agg')
+
+# NOTE right now this does not work for py3 because pickle files 
+
+import cPickle
+import matplotlib.pyplot as plt
+import numpy as np
+
+data_file = 'p38-Erlotinib-EF_mcmc-2017-05-05 13:16.pickle'
+###ABOVE INPUT PICKLE FILE NAME NEEDS MANUAL EDITTING
+
+with open(r'%s'%data_file,'rb') as my_file:
+    data = cPickle.load(my_file)
+
+import traceback
+import matplotlib
+colors = matplotlib.cm.viridis(np.linspace(0, 0.85, len(data.keys())))
+
+fig, axs = plt.subplots(len(data.keys())/2,2, figsize=(20, 80), facecolor='w', edgecolor='k')
+axs = axs.ravel()
+
+for i, key in enumerate(data.keys()):
+    try:
+        axs[i].plot(data['DeltaG'][0],data[key][0],color=colors[i],marker='.',linestyle = 'None');
+        axs[i].set_title('%s'%key,fontsize=16);
+        plt.tight_layout()
+    except Exception as e:
+        # There is often an error here for _state_, which it can't plot.
+        # It is fine, just uncomment the line below if you want to see it.
+        #print(traceback.print_exc())
+        print('%s has no [0]'%key)
+
+    plt.savefig('trace_p38-Erlotinib-EF_May5.png')
+    ###ABOVE OUTPUT PNG NAME NEEDS MANUAL EDITTING
+
+plt.clf()
+fig, axs = plt.subplots(len(data.keys())/2,2, figsize=(20, 80), facecolor='w', edgecolor='k')
+axs = axs.ravel()
+for i, key in enumerate(data.keys()):
+    try:
+        axs[i].plot(data[key][0],color=colors[i],marker='.',linestyle = 'None');
+        axs[i].set_title('%s'%key,fontsize=16);
+        plt.tight_layout()
+    except Exception as e:
+        # There is often an error here for _state_, which it can't plot.
+        # It is fine, just uncomment the line below if you want to see it.
+        #print(traceback.print_exc())
+        print('%s has no [0]'%key)
+
+    plt.savefig('DeltaGvall_p38-Erlotinib-EF_May5.png')
+    ###ABOVE OUTPUT PNG NAME NEEDS MANUAL EDITTING
+

